src/main/java/org/moltimate/moltimatebackend/service/AlignmentService.java

```diff
@@ -111,8 +111,8 @@ public ActiveSiteAlignmentResponse alignActiveSites(ActiveSiteAlignmentRequest a
      * @param motif:     motif that we search for in the structure
      * @return an alignment if one exists
      */
-    private Alignment alignActiveSites(Structure structure, Motif motif) {
-        Map<Residue, List<Group>> residueMap = motif.runQueries(structure, 1);
+    private Alignment alignActiveSites(Structure structure, Motif motif, int precisionFactor) {
```

Comment:
`precisionFactor` was merged into dev but fell out of the changes to method signature (this is what broke Travis)

Reply:
👍